demo/site/src/util/createSiteUrl.ts

```diff
@@ -0,0 +1,31 @@
+type CreateSiteUrlOptions = {
+    /**
+     * path with leading /
+     *
+     * Sample: "/test/to/my/page"
+     */
+    path: string;
```

Comment:
Should we validate this? I think this is something, which is common cause of errors.

Reply:
@dkarnutsch How do you want to validate this? In which direction do you think of?

Reply:
Do you think of checking and validating this directly in the function or in a separate unit test?

Reply:
added a check when the path does not start with a / that it will get added.